packages/compose/src/hooks/use-resize-observer/index.native.js

```diff
@@ -10,7 +10,7 @@ import { useState, useCallback } from '@wordpress/element';
 /**
  * Hook which allows to listen the resize event of any target element when it changes sizes.
  *
- * @return {Array} An array of {Element} `resizeListener` and {?Object} `sizes` with properties `width` and `height`
+ * @return {[JSX.Element, { width: number, height: number } | null]} An array of {Element} `resizeListener` and {?Object} `sizes` with properties `width` and `height`
```

Comment:
Why is the React Native version a completely different implementation of the web version?

Reply:
I'm asuming `react-resize-aware` doesn't work for react native 🤷‍♀️ In any case it's outside the scope of this PR. They have the exact same API FWIW.